lib/sport_ngin_aws_auditor/commands/audit.rb

```diff
@@ -11,9 +11,9 @@
   c.switch [:n, :no_tag], :desc => "Ignore all tags during audit"
   c.switch [:s, :slack], :desc => "Will print condensed version of audit to a Slack channel"
   c.switch [:z, :zone_output], :desc => "Will print the Missing RIs and Tagged instances with zones"
-  c.flag [:g, :ignore_instances_regex], :default_value => "kitchen, auto", :desc => "Ignore instances if an instance contains
+  c.flag [:g, :ignore_instances_strings], :default_value => "kitchen, auto", :desc => "Ignore instances if an instance contains
```

Comment:
The regex string is called a Pattern.  Lets change to to something like `ignored_instance_patterns` or `instance_ignore_pattern` to be more clear about what this argument is and does.